pages/docs/tooling/tooling-builtin.md

```diff
@@ -57,37 +57,35 @@ This prints the version information of preCICE, which consists of multiple semic
 3. Configuration options for MPI, PETSc, Python and some more.
 4. Compilation and linker flags used to build preCICE
 
-
 ## Configuration check
 
 {% version 2.4.0 %}
 This feature is available since version 2.4.0.
 {% endversion %}
 
-```
-$ binprecice check FILE [ PARTICIPANT [ COMMSIZE ] ]
+```bash
+binprecice check FILE [ PARTICIPANT [ COMMSIZE ] ]
 ```
 
 The `check` runs the preCICE configuration parsing and checking logic on the given configuration file.
 This will find the majority of the configuration mistakes without having to start a simulation.
 
 The basic usage is to simply check a configuration file.
 
-```
-$ binprecice check precice.xml
+```bash
+binprecice check precice.xml
 ```
 
 To enable more checks, additionally pass the name of one participant.
 This should be repeated for every defined participant in the simulation.
 
-```
-$ binprecice check precice.xml Fluid
+```bash
+binprecice check precice.xml Fluid
 ```
 
 To enable all available checks, you may additionally pass the communicator size of the Participant.
 This enables some niche checks, which should not be necessary in the vast majority of cases.
```

Comment:
While I do like that the "niche checks" tells me this is probably not something I need, I think that this is also a bit empty of additional information. Some examples / clarification in parentheses would help, otherwise every time I have a problem I would just enable everything without thinking.